git-rci.ps1

```diff
@@ -46,7 +46,10 @@ function Select-Branch {
 
         Write-Host ""
         Write-Host "Filter: '$filterText'" -ForegroundColor Cyan
-
+        Write-Host ""
+        Write-Host "'Use Arrow Keys to navigate. [Enter] to select/deselect the branch. [Esc] to complete"
+		
+        # If testing locally on Powershell, switch IncludeKeyDown => IncludeKeyUp
         $key = $host.UI.RawUI.ReadKey("NoEcho,IncludeKeyDown")
```

Comment:
Key presses so often have this kind of problem. I wonder if there's a different way that's more resilient 🤔